src/popup.js

```diff
@@ -14,7 +14,7 @@ optionsButton.addEventListener('click', () => {
 (async () => {
   const [currentTab] =
       await browser.tabs.query({active: true, currentWindow: true});
-  if (currentTab.url !== 'https://archiveofourown.org/works/new') {
+  if (!currentTab.url.match(/https:\/\/archiveofourown.org(\/collections\/.*)?\/works\/new/)) {
```

Comment:
I feel like having 2 conditions (new work || collections new work) for the predicate would be easier to read instead of the regex but I don't feel super strongly about that

Reply:
Sure, I don't mind making that change.